x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/action/EsqlQueryRequestTests.java

```diff
@@ -546,7 +597,7 @@ private StringBuilder paramsString(List<QueryParam> params, boolean hasParams) {
                     paramsString.append("\"");
                     paramsString.append(param.value());
                     paramsString.append("\"");
-                } else if (param.type().isNumeric() || param.type() == DataType.BOOLEAN || param.type() == DataType.NULL) {
+                } else if (param.type().isNumeric() || param.type() == DataType.BOOLEAN || param.type() == NULL) {
```

Comment:
for consistency, add static imports for the rest of the dataType

Reply:
Done